src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
+      && people.find((p1) => p1.mother === p.name))
```

Comment:
Nice work, you avoided making you line [longer than 80 characters](https://mate-academy.github.io/style-guides/javascript.html#1912avoid-having-lines-of-code-that-are-longer-than-80-characters-including-whitespace-note-perabove-long-strings-are-exempt-from-this-rule-and-should-not-be-broken-up)! But now, your arrow function body expression spans multiple lines and is a bit hard to read. Try moving it to separate line and enclosing in parentheses ([styleguide 8.3](https://mate-academy.github.io/style-guides/javascript.html#83-in-case-the-expression-spans-over-multiple-lines-wrap-it-in-parentheses-for-better-readability)). For example:

```suggestion
    ? people.filter((p) => (
        p.sex === 'f' && people.find((p1) => p1.mother === p.name)
      )
```